src/hashstore/hashstore_factory.py

```diff
@@ -28,8 +29,12 @@ def get_hashstore(self, hashstore_type):
         Raises:
             ValueError: If the given store_type is not supported.
         """
-        hashstore_type.lower()
-        if hashstore_type == "filehashstore":
-            return FileHashStore()
+
+        if hashstore_type.lower() == "filehashstore":
+            module_name = "hashstore.filehashstore.filehashstore"
+            class_name = "FileHashStore"
```

Comment:
Still has unwanted references to filehashstore in the factory. 

Given a module and classname as input (`hashstore_type`), why can't you simply assign them directly to `module_name` and `class_name`, without needing to know/validate what they contain? Presumably the call to `getattr()` will throw some sort of exception anyway, if the module/name are unrecognized - so why can't you just use that as your "validation" step?

Happy to jump on a call if I'm missing the point